stake-pool/js/src/buffer-layout.d.ts

```diff
@@ -1,16 +0,0 @@
-declare module 'buffer-layout' {
```

Comment:
`tsc` is failing without this -- could add it back in?  It might also be better for the `build` step to use `&&` instead of `;` to catch this in the future, ie:
```
"build": "npm run clean && tsc && cross-env NODE_ENV=production rollup -c",
```